src/modules/displaymanager/main.py

```diff
@@ -147,19 +177,37 @@ def set_autologin(username, displaymanagers, default_desktop_environment, root_m
                 else:
                     gdm_conf.write('AutomaticLoginEnable=False\n')
 
-        if do_autologin and os.path.exists("{!s}/var/lib/AccountsService/users".format(root_mount_point)):
-            os.system("echo \"[User]\" > {!s}/var/lib/AccountsService/users/{!s}".format(root_mount_point, username))
+        if do_autologin and os.path.exists(
+                "{!s}/var/lib/AccountsService/users".format(root_mount_point)):
+            os.system(
+                "echo \"[User]\" >",
```

Comment:
Not particularly related to this review, but it seems kind of dodgy to me to involve os.system() to do what is basically

```
with open("{!s}/var/lib/AccountsService/users/{!s}".format(root_mount_point, username), "w") as f:
    f.write("[User]\n")
```

Unless there's some permissions-trickery going on here, I think this whole module would benefit from a re-write to clean up the os.system() calls.